### PR TITLE
OCPBUGS-24214: Add ownership annotations

### PR DESCRIPTION
**Pull request by vrutkovs**

### CODE REVIEW

File: pkg/controller/pki/pki_controller.go [hunk 1]
```diff
@@ -194,6 +194,7 @@ func newPKI(config *netopv1.OperatorPKI, clientset *kubernetes.Clientset, mgr ma
 		certrotation.RotatedSigningCASecret{
 			Namespace:     config.Namespace,
 			Name:          config.Name + "-ca",
+			JiraComponent: "Networking / cluster-network-operator",
```
Comment (kyrtapz) [inline pkg/controller/pki/pki_controller.go:197]: Please use `names.ClusterNetworkOperatorJiraComponent` instead

**Comment by vrutkovs** [comment]
Updated to target 4.16. We don't need to backport it to 4.14 or 4.15 (at least yet)

File: bindata/network/ovn-kubernetes/managed/007-certificate.yaml [hunk 2]
```diff
@@ -8,6 +8,8 @@ metadata:
   annotations:
     network.operator.openshift.io/cluster-name:  {{.ManagementClusterName}}
     network.operator.openshift.io/copy-from: "default/openshift-ovn-kubernetes/ovn-cert"
+    annotations:
```
Comment (kyrtapz) [inline bindata/network/ovn-kubernetes/managed/007-certificate.yaml:11]: That's wrong, please remove  `annotations:`

Comment (kyrtapz) [inline bindata/network/ovn-kubernetes/managed/007-certificate.yaml:11]:
```suggestion
    openshift.io/owning-component: "Networking / cluster-network-operator"
```

Comment (vrutkovs) [reply bindata/network/ovn-kubernetes/managed/007-certificate.yaml]: Oops, bad rebase. Fixed

**Comment by kyrtapz** [comment]
CI was broken
/retest

**Pull request merged**